enhancements/olm-deprecate-copied-csvs.md

```diff
@@ -0,0 +1,155 @@
+**Mitigation:** N/A
+
+#### OpenShift Console Integration
+
+**Risk:** The OpenShift Console uses Copied CSVs in the rendering of certain pages.
+**Mitigation:** Consult Console maintainers prior to deprecating Copied CSVs.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. Is it acceptible to require a CVO override -- i.e. block cluster upgrades -- to disable Copied CSVs in the short-term (before the feature is completely deprecated)?
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+- Deprecation of Copied CSVs announced; removal within two minor releases
+- Feature toggle introduced, with Copied CSVs enabled by default 
+- End user documentation, relative API stability
+- Sufficient test coverage
+
+#### Tech Preview -> GA
+
+- Copied CSVs disabled by default
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Feature toggle removed
+
+### Upgrade / Downgrade Strategy
+
+If applicable, how will the component be upgraded and downgraded? Make sure this
+is in the test plan.
+
+Upgrade expectations:
+- Each component should remain available for user requests and
+  workloads during upgrades. Ensure the components leverage best practices in handling [voluntary disruption](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/). Any exception to this should be
+  identified and discussed here.
+- Micro version upgrades - users should be able to skip forward versions within a
+  minor release stream without being required to pass through intermediate
+  versions - i.e. `x.y.N->x.y.N+2` should work without requiring `x.y.N->x.y.N+1`
+  as an intermediate step.
+- Minor version upgrades - you only need to support `x.N->x.N+1` upgrade
+  steps. So, for example, it is acceptable to require a user running 4.3 to
+  upgrade to 4.5 with a `4.3->4.4` step followed by a `4.4->4.5` step.
+- While an upgrade is in progress, new component versions should
+  continue to operate correctly in concert with older component
+  versions (aka "version skew"). For example, if a node is down, and
+  an operator is rolling out a daemonset, the old and new daemonset
+  pods must continue to work correctly even while the cluster remains
+  in this partially upgraded state for some time.
+
+## Implementation History
+
+- N/A
+
+## Drawbacks
+
+- The Operator API is not a "drop-in-place" for Copied CSVs
+- There is no equivalent to namespace scoped CSV `List` requests for `Operator` resources; i.e. users must perform individual `Get` requests for each `Operator` they want to inspect, and must comb through `Operator` status to determine if an operator can act on a target namespace.
+
+## Alternatives
+
+### Virtual InstallOperator API
+A "virtual resource" served by an aggregated API -- much like `PackageManifest` -- that provides a namespaced, CSV-like, view of the operators configured to interact with a given namespace. Such a resource would be a direct drop-in-place for a Copied CSV.
```

Comment:
Would it be possible to augment why we're not proposing this alternative here, even if it's as straight-forward as we want a low-cost way of relieving the scalability issues around the combination copied CSVs + AllNamespaced-scoped Operator in highly dense cluster environments, as we'd like to eventually deprecate the OperatorGroup resource as well, and don't want the maintenance burden of another deprecation of this newly introduced aggregated API>